x/ecocredit/server/core/query_params_test.go

```diff
@@ -3,26 +3,55 @@ package core
 import (
 	"testing"
 
-	"github.com/golang/mock/gomock"
 	"gotest.tools/v3/assert"
 
-	"github.com/cosmos/cosmos-sdk/types"
+	basev1beta1 "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	basketv1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/basket/v1"
+	ecocreditv1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1"
 	"github.com/regen-network/regen-ledger/x/ecocredit/core"
 )
 
 func TestQuery_Params(t *testing.T) {
 	t.Parallel()
 	s := setupBase(t)
-	gmAny := gomock.Any()
-	s.paramsKeeper.EXPECT().GetParamSet(gmAny, gmAny).SetArg(1, core.Params{
-		CreditClassFee:       types.NewCoins(types.NewInt64Coin("foo", 30)),
-		AllowedClassCreators: []string{s.addr.String()},
-		AllowlistEnabled:     false,
+
+	err := s.stateStore.AllowedClassCreatorTable().Save(s.ctx, &ecocreditv1.AllowedClassCreator{
+		Address: s.addr,
+	})
+	assert.NilError(t, err)
+
+	err = s.stateStore.AllowListEnabledTable().Save(s.ctx, &ecocreditv1.AllowListEnabled{
```

Comment:
```suggestion
	err = s.stateStore.AllowListEnabledTable().Insert(s.ctx, &ecocreditv1.AllowListEnabled{
```

Reply:
We don't have Insert method for singleton tables.